### ISSUE TITLE
What is the number of pseudo images generated by Stable Diffusion for GORS fine-tuning? 

### ISSUE THREAD

**Issue by Yongssss** [Karine-Huang/T2I-CompBench#1]
Thanks for your excellent work. I have a question, that is, what is the number of pseudo images generated by Stable Diffusion for GORS fine-tuning?

**Comment by Karine-Huang** [comment]
Thank you for your kind words! In regards to your question, for each prompt in the training set used in the Stable Diffusion for GORS fine-tuning, a total of 90 pseudo images are generated. Considering that there are 700 prompts, total number of pseudo images generated would be 700 prompts multiplied by 6 (the number of prompts in each sub-category) multiplied by 90 (the number of pseudo images generated per prompt), resulting in a total of 378,000 images (700 * 6 * 90).

**Comment by Yongssss** [comment]
Thanks for your detailed reply. Do you have a plan to release the dataset consisting of pseudo images?

**Comment by Karine-Huang** [comment]
We will release the code in order to regenerate the pseudo images (including the seed and batch size we use) in the following days.
